### ISSUE TITLE
Checkbox icon in the wrong position

### ISSUE THREAD

**Issue by herlon214** [avocode/nachos-ui#2]
I don't know if this is only on my browser but the icon seems to be in the wrong position.
https://avocode.com/nachos-ui/docs/#!/Showcase/Checkbox

![Printscreen](http://image.prntscr.com/image/e79fca12283b43ffa136a13fc8ab1e0e.png)

**Comment by vojtatranta** [comment]
it's okey in my Chrome, anyway how it looks on mobile matters more :)

**Comment by petrbrzek** [comment]
@herlon214 Thanks for the report. What browser do you use?

**Comment by herlon214** [comment]
I agree with you @vojtatranta , it was just an observation. But someone can think that in the mobile is the same thing maybe.

@petrbrzek my browser is Firefox 51.0.1

**Comment by yudori** [comment]
yep! same issue here as well Firefox 51.0.1 on Ubuntu

**Comment by petrbrzek** [comment]
I will look at it but I'm not sure if I can fix it as this depends on React Native for Web.

**Comment by petrbrzek** [comment]
So we depend on React Native for Web to fix this [issue](https://github.com/necolas/react-native-web/issues/241).

**Comment by necolas** [comment]
FYI your checkbox/radio components aren't accessible on web. I'd suggest building them more like how `Switch` is implemented in `react-native-web`.
